<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>685</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1082</width>
    <height>685</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1082</width>
    <height>685</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LoginWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">#centralWidget{
border-image: url(:/image/login/login_bj.png);
}
</string>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>611</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/login/图片2.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>273</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/login/farsight.png);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Name_lineEdit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>360</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/login/图片4.png);
font: 12pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>请输入账号</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>365</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/login/登录 (4).png);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Password_lineEdit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>435</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/login/图片4.png);
font: 12pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>请输入密码</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>440</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/login/登录页面-密码.png);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>390</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/login/图片4.png);
font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>记住密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoginButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>453</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/image/login/图片4.png);
font: 10pt &quot;微软雅黑&quot;;
}
QPushButton:pressed{
font: 9pt &quot;微软雅黑&quot;;
}</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>390</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/login/loging_button_bj.png);
font: 9pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="forgetPasswprdpushButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>520</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 9pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);
border-image: url(:/image/login/图片4.png);
text-decoration: underline;
}
QPushButton:pressed{
font: 8pt &quot;微软雅黑&quot;;
}</string>
    </property>
    <property name="text">
     <string>忘 记 密 码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetPasswordpushButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>520</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 9pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);
border-image: url(:/image/login/图片4.png);
text-decoration: underline;
}
QPushButton:pressed{
font: 8pt &quot;微软雅黑&quot;;
}</string>
    </property>
    <property name="text">
     <string>重 置 密 码</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1082</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>功能</string>
    </property>
    <addaction name="serverAction"/>
    <addaction name="registeraction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="registeraction">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/image/login/登录7.png</normaloff>:/image/login/登录7.png</iconset>
   </property>
   <property name="text">
    <string>注册账号</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="serverAction">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/image/login/服务器IP申请.png</normaloff>:/image/login/服务器IP申请.png</iconset>
   </property>
   <property name="text">
    <string>设置服务器地址</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
